include/fastboot.jinja2

```diff
@@ -18,6 +18,13 @@
 {% set pre_os_command = pre_os_command|default(true) %}
 {% set auto_login = auto_login|default(true) %}
 {% set boot_method = boot_method|default("fastboot") %}
+{% set deploy_download_timeout = deploy_download_timeout|default(target_deploy_timeout) %}
+
+{% DEPLOY_TARGET == "downloads" %}
+{% set calc_target_deploy_timeout = deploy_download_timeout + target_deploy_timeout %}
+{% else %}
+{% set calc_target_deploy_timeout = target_deploy_timeout %}
+{% endif %}
```

Comment:
http://ix.io/3WW2 I think this should be what we should do.

Reply:
this is what you shared me before: http://ix.io/3Wcd.
And I am not sure why fastboot_deploy_timeout  needs to be added in your new patch, any explanation on it?

Reply:
so I've crafted a third patch, that I think makes more sense than the others.
Lets see what you think http://ix.io/3XsQ

Reply:
@roxell could you please submit your patch via another PR?